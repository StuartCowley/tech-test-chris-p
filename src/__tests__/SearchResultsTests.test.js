import React from "react";
import {render} from "@testing-library/react";
import SearchResults from "../components/SearchResultsComponent";

describe("search results component", () => {
    const {asFragment} = render(<SearchResults/>);

    it("search results component renders correctly", () => {
        expect(asFragment).toMatchSnapshot();
    });
});