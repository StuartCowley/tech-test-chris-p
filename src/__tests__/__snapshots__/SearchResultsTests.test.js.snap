// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`search results component search results component renders correctly 1`] = `[Function]`;
